tests/unittests/core/io/interactive_commands/test_branching_prompt.py

```diff
@@ -63,14 +63,6 @@ def conflicts(new_dimension_conflict, new_cat_dimension_conflict,
     return conflicts
 
 
-@pytest.fixture
-def conflicts_exp_name(conflicts, failing_exp_name_conflict, experiment_name_conflict):
-    """Return a list of conflicts with an ExperimentNameConflict that fails."""
```

Comment:
Why it isn't needed anymore? Because all exp-name now fails?

Reply:
Because it was a workaround for the fact that the `ExperimentNameConflict` inside the original `conflicts` did not fail anymore. So I just replaced the current passing `ExperimentNameConflict` with the one that fails which now invalidates the existence of that fixture.